tests/unit/test_plasma_profiles.py

```diff
@@ -291,15 +291,15 @@ class PlasmaProfilesParam(NamedTuple):
             gradient_length_te=0.0,
             rminor=2.9264516129032256,
             expected_prn1=0.45623618297262686,
-            expected_ten=14.521871327399182,
-            expected_tin=14.521871327399182,
+            expected_ten=14.52233022043558,
+            expected_tin=14.52233022043558,
             expected_alphap=2.4500000000000002,
-            expected_te0=27.369013322953624,
-            expected_p0=868071.46874220832,
-            expected_pcoef=1.1110842637642833,
+            expected_te0=27.370104119511087,
+            expected_p0=868106.0658743214,
```

Comment:
Again, why so much bigger than the others

Reply:
^ see above